src/server/scripts/Outland/TempestKeep/Eye/boss_kaelthas.cpp

```diff
@@ -939,6 +942,15 @@ class boss_thaladred_the_darkener : public CreatureScript
                 else
                     Silence_Timer -= diff;
 
+                //Rend_Timer
+                 if (Rend_Timer <= diff)
+                 {
+                    DoCastVictim(SPELL_REND);
+                    Rend_Timer = 8000;
```

Comment:
Sniffs show it being cast at 4 seconds always. But don't change it now, leave it as it is.